doc/nrf/software_maturity.rst

```diff
@@ -7,28 +7,41 @@ Software maturity levels
    :local:
    :depth: 2
 
-Following are the different software maturity levels shown for the different features:
+|NCS| supports its various features and components at different levels of software maturity.
 
-* Supported - Features that are maintained and are suitable for product development.
-* Experimental - Features that can be used for development but not recommended for production.
+The following are the three levels used to classify the software maturity of each feature and component:
 
-   * Feature is incomplete in functionality or verification, and can be expected to change in future releases. The feature is made available in its current state though the design and interfaces can change between release tags. The feature will also be labelled with "EXPERIMENTAL" in Kconfig files to indicate this status. Build warnings will be generated to indicate when features labelled EXPERIMENTAL are included in builds unless the Kconfig option :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` is disabled.
+* *Supported* - The feature or component is maintained and is suitable for product development.
+* *Not supported* - The feature or component does not work.
+* *Experimental* - The feature can be used for development but it is not recommended for production.
 
-* Not supported - The feature or component will not work if not supported.
+  This means that the feature is incomplete in functionality or verification and can be expected to change in future releases.
```

Comment:
This doesn't work visually. It's really hard to see that this text is for "Experimental". Could we use a table or something more visible?

![image](https://user-images.githubusercontent.com/12450381/168816929-844c2679-ea72-4d49-982c-d81b79a06d84.png)

Reply:
I agree this version is not perfect, but it still looked better than before. I'll try to come up with a better idea.

Reply:
Try using the definition list:

```suggestion
Supported
   The feature or component is maintained and is suitable for product development.

Not supported
   The feature or component does not work.

...
```

Reply:
Right, fixed